src/index.js

```diff
@@ -0,0 +1 @@
+// DELETE ME
```

Comment:
what is this file for?

Reply:
It's temporary just so the `src` directory got committed.

Reply:
I think this is simply the start file
```suggestion
<Insert code>
```